simphony/io/tests/test_file_mesh.py

```diff
@@ -0,0 +1,686 @@
+            puuids.append(puuid)
+
+        cells = [
+            Cell(puuids[0:4]),
+            Cell(puuids[1:5]),
+            Cell(puuids[2:6])
+            ]
+
+        for cell in cells:
+            cuuid = self.mesh.add_cell(cell)
+            cuuids.append(cuuid)
+
+        icells = self.mesh.iter_cells([cuuids[0], cuuids[2]])
+
+        source_id = [cuuids[0], cuuids[2]]
+        icells_id = [cell.uuid for cell in icells]
+
+        self.assertItemsEqual(source_id, icells_id)
+
+    def test_update_point(self):
+        """ Check that a point can be updated correctly
+
+        """
+
+        puuids = []
+
+        points = [
+            self.points[0],
+            ]
+
+        for point in points:
+            puuid = self.mesh.add_point(point)
+            puuids.append(puuid)
+
+        point_ret = self.mesh.get_point(puuids[0])
+        point_ret.coordinates = [-1.0, -1.0, -1.0]
+        self.mesh.update_point(point_ret)
+
+        point_upd = self.mesh.get_point(puuids[0])
+
+        self.assertItemsEqual(point_upd.coordinates, point_ret.coordinates)
+
+    def test_update_edge(self):
+        """ Check that an edge can be updated correctly
+
+        """
+
+        puuids = []
+        euuids = []
+
+        points = [
+            self.points[0],
+            self.points[1],
+            self.points[2]
+            ]
+
+        for point in points:
+            puuid = self.mesh.add_point(point)
+            puuids.append(puuid)
+
+        edge = Edge(puuids[0:2])
+
+        euuid = self.mesh.add_edge(edge)
+        euuids.append(euuid)
+
+        edge_ret = self.mesh.get_edge(euuids[0])
+        edge_ret.points[1] = puuids[2]
+        self.mesh.update_edge(edge_ret)
+
+        edge_upd = self.mesh.get_edge(euuids[0])
+
+        self.assertItemsEqual(edge_upd.points, edge_ret.points)
+
+    def test_update_face(self):
+        """ Check that a face can be updated correctly
+
+        """
+
+        puuids = []
+        fuuids = []
+
+        points = [
+            self.points[0],
+            self.points[1],
+            self.points[2],
+            self.points[3]
+            ]
+
+        for point in points:
+            puuid = self.mesh.add_point(point)
+            puuids.append(puuid)
+
+        face = Face(puuids[0:3])
+
+        fuuid = self.mesh.add_face(face)
+        fuuids.append(fuuid)
+
+        face_ret = self.mesh.get_face(fuuids[0])
+        face_ret.points[2] = puuids[3]
+        self.mesh.update_face(face_ret)
+
+        face_upd = self.mesh.get_face(fuuids[0])
+
+        self.assertItemsEqual(face_upd.points, face_ret.points)
+
+    def test_update_cell(self):
+        """ Check that a cell can be updated correctly
+
+        """
+
+        puuids = []
+        cuuids = []
+
+        points = [
+            self.points[0],
+            self.points[1],
+            self.points[2],
+            self.points[3],
+            self.points[4]
+            ]
+
+        for point in points:
```

Comment:
suggestion (it is not a real problem):  
Using list comprehantions is a common python idiom.
`puuids = [self.mesh.add_point(point) for point in points] `